File: app/src/main/java/com/zionhuang/music/utils/SuggestionProvider.java
package com.zionhuang.music.utils;

import android.content.SearchRecentSuggestionsProvider;

public class SuggestionProvider extends SearchRecentSuggestionsProvider {
    public final static String AUTHORITY = "com.zionhuang.music.utils.SuggestionProvider";
    public final static int MODE = DATABASE_MODE_QUERIES;

    public SuggestionProvider() {
        setupSuggestions(AUTHORITY, MODE);
    }
}

File: app/src/main/java/com/zionhuang/music/utils/SuggestionProvider.kt
package com.zionhuang.music.utils

import android.content.SearchRecentSuggestionsProvider
import com.zionhuang.music.BuildConfig

class SuggestionProvider : SearchRecentSuggestionsProvider() {
    companion object {
        const val AUTHORITY = "${BuildConfig.APPLICATION_ID}.utils.SuggestionProvider"
        const val MODE = DATABASE_MODE_QUERIES
    }

    init {
        setupSuggestions(AUTHORITY, MODE)
    }
}